cmd/wrestic/main.go

```diff
@@ -134,59 +134,73 @@ func run(resticCLI *restic.Restic, mainLogger logr.Logger) error {
 	}
 
 	if *archive {
-		commandRun = true
+		dontBackup = true
```

Comment:
Personally I'd find this name a little easier on my thoughts in my brain :)
```suggestion
		skipBackup = true
```